File: Composicion_y_Agregacion/Ejercicio1/Python/Habitacion.py

class Habitacion:
    def __init__(self, nombre, tamaño):
        self.nombre = nombre
        self.tamaño = tamaño
        
    def mostrar_casa(self):
        print(f"\nHabitación: {self.nombre}")
        print(f"Tamaño: {self.tamaño} m²")
        
    def get_nombre(self):  
         return self.nombre
    def get_tamaño(self):  
         return self.tamaño

File: Composicion_y_Agregacion/Ejercicio1/Java/Habitacion.java
package Composicion_y_Agregacion.Ejercicio1.Java;

public class Habitacion {
    private String nombre;
    double tamaño;

    public Habitacion (String nombre, double tamaño) {
        this.nombre = nombre;
        this.tamaño = tamaño;
    }

    void mostrar_casa() {
        System.out.println("\nHabitación: " + nombre); 
        System.out.println("Tamaño: " + tamaño + " m2");

    }

    String getNombre() { return nombre; } 
    double getTamaño() { return tamaño; }
    }